openmls/src/test_utils/test_framework/mod.rs

```diff
@@ -240,11 +240,18 @@ impl MlsGroupTestSetup {
             CodecUse::StructMessages => MlsMessageIn::from(message.clone()),
         };
         let clients = self.clients.read().expect("An unexpected error occurred.");
-        // Distribute message to all members.
+        // Distribute message to all members, except to the sender in the case of application messages
```

Comment:
Shouldn't this be a problem for handshake messages as well?

Reply:
That's true, but the test framework is only used with plaintext messages currently. It was not adapted to the pending commit logic in #659, so it currently cannot do ciphertext messages I guess?

Reply:
I'm pretty sure it was adapted. It could be that no test actually configures a group to encrypt handshake messages. I'll file a follow up to investigate.